### PR TITLE
Added highlight for the row with the current local time

### LINKED ISSUE #12: Highlight row with the current local time

### PR DESCRIPTION
**Pull request by Badboy-16**
Fixed #12 

Using the text formatting tool in `rich`, I added some code to highlight the row with current hour in red. Please let me know if other colours/formats are preferred and I may change the highlight. :)

### CODE REVIEW

File: timezone_converter/comparison_view.py [hunk 1]
```diff
@@ -53,11 +53,21 @@ def _build_table(self) -> Table:
...
 
         fmt = '%Y-%m-%d %H:%M'
+
+        current_hour = datetime.now().hour
         for hour in range(24):
-            columns = [
-                (midnight + timedelta(hours=hour)).strftime(fmt)
-                for midnight in self.midnights
-            ]
+            if hour != current_hour:
```
Comment (ibLeDy) [inline timezone_converter/comparison_view.py:59]: I have some suggestions:

- I prefer that we use a default value and then check if the hour matches to then change the value
- We can use `f-strings`
- My color suggestion would be blue, as red is usually reserved for errors, and the opposite for green (which we may end up using later for other features)

This is the how the code would end up looking:

```py
columns = [
    (midnight + timedelta(hours=hour)).strftime(fmt)
    for midnight in self.midnights
]
if hour == current_hour:
    columns = [
        f'[blue]{(midnight + timedelta(hours=hour)).strftime(fmt)}[/blue]'
        for midnight in self.midnights
    ]

```

**Review by ibLeDy — CHANGES_REQUESTED** [review]
I have some suggestions, but your contribution looks good!

File: timezone_converter/comparison_view.py [hunk 2]
```diff
@@ -53,11 +53,21 @@ def _build_table(self) -> Table:
...
+                    (midnight + timedelta(hours=hour)).strftime(fmt)
+                    for midnight in self.midnights
+                ]
+            else:
+                columns = [
+                    '[red]'
+                    + (midnight + timedelta(hours=hour)).strftime(fmt)
+                    + '[/red]'
+                    for midnight in self.midnights
+                ]
```
Comment (ibLeDy) [inline timezone_converter/comparison_view.py:70]: Suggestions:

- I prefer that we use a default value and then check if the hour matches to then change the value
- We can use `f-strings`
- My color suggestion would be blue, as red is usually reserved for errors, and the opposite for green (which we may end up using later for other features)

This is the how the code would end up looking:

```py
columns = [
    (midnight + timedelta(hours=hour)).strftime(fmt)
    for midnight in self.midnights
]
if hour == current_hour:
    columns = [
        f'[blue]{(midnight + timedelta(hours=hour)).strftime(fmt)}[/blue]'
        for midnight in self.midnights
    ]

```

**Review by ibLeDy — CHANGES_REQUESTED** [review]

File: timezone_converter/comparison_view.py (hunk 2 above)
Comment (Badboy-16) [reply timezone_converter/comparison_view.py]: Thanks for the suggestion! I have pushed a new commit to update the code.

Comment (ibLeDy) [reply timezone_converter/comparison_view.py]: Looks good, thanks!

**Comment by ibLeDy** [comment]
Thanks for your contribution, I will publish a new release to PyPi with this changes!

**Pull request merged**